### DISCUSSION TITLE
How to remove the limit on image resizing

### DISCUSSION THREAD

**Discussion by jsericksk** [burhanrashid52/PhotoEditor#404] [Q&A] [answered]
There appears to be a limit to where images can be resized with drag and drop. Is there any way to remove this limit so that the image can be resized as small as possible?

In the example below I used a sticker from the sample app, but the same behavior happens with any image.

Thanks for the library, by the way. I'm using it in an app I made recently. (can be checked [here](https://play.google.com/store/apps/details?id=com.kproject.memezim))

![ezgif-2-32f644638bdd](https://user-images.githubusercontent.com/36176569/136487176-3855f6e5-e9b2-4ca6-a14c-d55d95833ec7.gif)

**Comment by burhanrashid52** [comment, marked as answer]
Thanks for creating this discussion. Yes, currently it's not supported. We might need to work [ScaleGestureDetector](https://github.com/burhanrashid52/PhotoEditor/blob/master/photoeditor/src/main/java/ja/burhanrashid52/photoeditor/ScaleGestureDetector.java). Any PR is welcome.

**Comment by burhanrashid52** [comment]
Also, I am keeping the list in README for the app that is using PhotoEditor. https://github.com/burhanrashid52/PhotoEditor#who-is-using-photoeditor
I can add your app if you wish.

**Reply by jsericksk** [reply]
Yes, you can add, I would appreciate it.

**Comment by jsericksk** [comment]
As a workaround, I've added a fixed size of 100 dp in the [ImageView's](https://github.com/burhanrashid52/PhotoEditor/blob/master/photoeditor/src/main/res/layout/view_photo_editor_image.xml) width and height, so the image is already inserted with a minimal size and resizable without much limitation.
